local-infrastructure/README.md

```diff
@@ -27,7 +27,7 @@ Application setup details and more are in the [Developer Guide](# https://bcgov.
 
 ### Quick Run Steps
 
-1. Sign up at the [GETOK Common Service Onboarding page](https://getok.pathfinder.gov.bc.ca/getok/about).
+1. Sign up at the [GETOK Common Service Onboarding page](https://getok.nrs.gov.bc.ca/getok/about).
```

Comment:
Path is incorrect - should be https://getok.nrs.gov.bc.ca/app/about

Reply:
👍